### PR TITLE
Regression/SDK breaks if initialised as a modal

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
If initialised in modal mode, the SDK breaks and throws an error. UI tests for modal scenario in recent PRs are failing because of this.

# Solution
Use object destructuring on the props object that is now passed in to Modal component as a parameter instead of `this.props`.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [n/a] Has the CHANGELOG been updated? - dev regression, bug is not in production
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1171-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1171-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1171-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Modal/index.js [hunk 1]
```diff
@@ -12,43 +12,42 @@ const MODAL_ANIMATION_DURATION = getCSSMilisecsValue(
...
 const Wrapper = ({ children }) => wrapWithClass(style.inner, children)
 
-const Modal = () => {
-  const {
-    translate,
-    isFullScreen,
-    containerId,
-    containerEl,
-    shouldCloseOnOverlayClick,
-  } = this.props
```
Comment (ethan605) [inline src/components/Modal/index.js:22]: Seems that it's my bad from #1157: https://github.com/onfido/onfido-sdk-ui/pull/1157/files#diff-b17c47ef8137161ae8a178243850f1c4R22. So funny that we all didn't catch it 😅

**Review by ethan605 — APPROVED** [review]
Thanks for spotting & fixing this!

File: src/components/Modal/index.js (hunk 1 above)
Comment (kopijunkie) [reply src/components/Modal/index.js:22]: Yeah, I was looking at it thinking "I can't believe I missed this" 🙈

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1171-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1171-pr-onfido-sdk-ui-onfido.surge.sh

**Review by stefaniacardenas — APPROVED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1171-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1171-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by ethan605** [comment]
Close in favour of #1174

**Pull request closed without merging**